api/imapsync/read

```diff
@@ -50,20 +50,22 @@ if ($cmd eq 'list') {
             'displayname' => $1,
             'type' => 'imapsync'
         };
-        my $password = '';
-        if ( -f "/var/lib/nethserver/secrets/imapsync-$_") {
-            $password = NethServer::Password::store('imapsync-'.$_);
-        }
+
+        my $password;
+        local $/; #Enable 'slurp' mode
+        open my $fh, "<", '/var/lib/nethserver/imapsync/secrets/'.$_;
+        $password = <$fh>;
+        close $fh;
```

Comment:
Wrapping into a `do {}` block avoids altering the `$/` value for the rest of the code.

```suggestion
        open my $fh, "<", '/var/lib/nethserver/imapsync/secrets/'.$_;
        # Read until EOF 'slurp' mode
        my $password = do { local $/; <$fh> };
        close $fh;
```

See https://www.perl.com/article/21/2013/4/21/Read-an-entire-file-into-a-string/
Even more compact solutions exist around! I've seen also the `open()` inside the `do {}` block, that should make the `close()` implicit!

Reply:
done